core/templates/pages/exploration-player-page/services/question-player-engine.service.spec.ts

```diff
@@ -674,21 +580,29 @@ describe('Question player engine service ', () => {
     questionPlayerEngineService.init(
       [singleQuestionBackendDict], successHandler, failHandler);
 
-    expect(alertsServiceSpy).toHaveBeenCalledWith('Expression parsing error.');
+    expect(alertsServiceSpy).toHaveBeenCalledWith(
+      'Question name should not be empty.');
   });
 
   it('should not load questions if there are no questions', () => {
     let successHandler = jasmine.createSpy('success');
     let failHandler = jasmine.createSpy('fail');
 
+    let alertsServiceSpy =
+    spyOn(alertsService, 'addWarning').and.returnValue();
```

Comment:
Format this correctly -- break after '(', not '='. Also if you break to a newline, indent by 2.

/cc @Hudda is this tracked in the linter? It seems like something basic.

Reply:
done.

Reply:
@seanlip added to the #8423. Thanks